Dev/config/strings.js

```diff
@@ -130,6 +130,9 @@ export default {
     Whoops: 'Whoops',
     AddStudents: 'Add Students',
     JoinMyClass: 'Join my class on QuranConnect. Here is the class code: ',
+    AddYourStudents: 'Add Your Students',
+    Or: 'Or',
```

Comment:
This is fine for now, we eventually need to enhance the UX and the messaging a bit. Someone can take a look at how other apps offer offline/online registration and mimmic their UI there where it makes sense.